src/scenario.jl

```diff
@@ -478,9 +480,10 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
     α = 0.95
     decay = α .^ (1:Int64(param_set("plan_horizon"))+1)
 
-    # Years at which to reassess seeding site selection
+    # Years at which seeding/shading/fogging occurs
     seed_decision_years = fill(false, tf)
     shade_decision_years = fill(false, tf)
+    fog_decision_years = fill(false, tf)
```

Comment:
Is the modified comment correct, or has the conceptualization changed?

My understanding was that:

- `*_freq` controls how often location selection occurs (every year, every 3rd year, etc)
- `*_decision_years` then speak to which years in the simulation location selection occurs
- It does not relate to years interventions are deployed

If `fog_freq` is 3, it re-evaluates where interventions should occur every third year.
So `fog_decision_years` holds [false, false, true, false, false, true, false, false, true, ...]

Interventions still occur every year, it's just that _where_ interventions occur changes every third year.

Is this not correct?

Reply:
You're correct, I think maybe this comment was originally for `in_seed_years` , `in_shade_years` etc. definitions. I've moved it to where there are defined and added a comment to describe `*_decision_years`